www/biko00/sp2/js/main.js

```diff
@@ -0,0 +1,225 @@
+$(document).ready(function () {
+  $('#search-recipes').click(function () {
+
+    var cuisine = $('#cuisine').val();
+    var diet = $('#diet').val();
+    var intolerances = $('#intolerances').val();
+    var maxReadyTime = $('#maxReadyTime').val() || '30';
+    $('#recipe-container').html('');
+    toggleVisibility('recipe-spinner', true);
+    $.ajax({
+      url: 'https://api.spoonacular.com/recipes/complexSearch',
+      type: 'GET',
+      dataType: 'json',
+      data: {
+        apiKey: 'aa8b5e5027e04fac90bd86cbd40ef03a',
+        cuisine: cuisine,
+        diet: diet,
+        intolerances: intolerances,
+        maxReadyTime: maxReadyTime,
+        addRecipeInformation: true,
+        number: 3
+      },
+      success: function (response) {
+        displayRecipes(response.results);
+        toggleVisibility('recipe-spinner', false);
+      },
+      error: function (error) {
+        console.log('Error:', error);
+        toggleVisibility('recipe-spinner', false);
+      }
+    });
+
+  });
+
+  $('#show-liked-recipes').click(function () {
+    var likedRecipesContainer = $('#liked-recipes-container');
+    if (likedRecipesContainer.is(':visible')) {
+      likedRecipesContainer.hide();

```

Comment:
to co nema byt videt na strance by nemel byt ani v DOM, existuji vyjimky, nicmene tohle neni jedna z nich, takze bud si ty liked recepty zobrazujte v modalnim okne, nebo budete prepinat mezi dvema sekcemi a renderovat je